serving-operator/controllers/serve_httpclient.go

```diff
@@ -0,0 +1,113 @@
+package controllers
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/util/json"
+	"net/http"
+
+	rayv1alpha1 "github.com/ray-project/kuberay/api/v1alpha1"
+)
+
+var (
```

Comment:
Is there any way to force these endpoints to stay consistent with the Python code?

Reply:
If we can get these values from some config repo, then python and golang can sync.